service/worker/scanner/scanner.go

```diff
@@ -243,8 +247,14 @@ func (s *Scanner) startWorkflow(
 	cancel()
 
 	if cadence.IsWorkflowExecutionAlreadyStartedError(err) {
+		s.zapLogger.Error("Workflow had already started", zap.String("workflowType", workflowType), zap.Error(err))
```

Comment:
better to use predefined logs tags instead of repeating same tag key 
- `tag.WorkflowType`
- `tag.WorkflowTaskListName`
- we don't have predefined one for "workerTaskListNames" so it's probably fine to define it inline in this file